setup.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import absolute_import
 
-from glob import glob
 import os
+from glob import glob
```

Comment:
Imports should be in the alphabetical order.

Reply:
actually our style guide (google's) puts `from` imports after `import` imports